tests/other_model_test/test_cluster_sampler.ipynb

```diff
@@ -2240,6 +2240,36 @@
    "source": [
     "euclidean_distance_kernels_v(cluster_centers)"
    ]
+  },
+  {
```

Comment:
I'm confused. This notebook seems to be full of output and error messages. What exactly are we doing here?

Reply:
old code, this will be deleted